llvm/tools/repo2obj/R2OELFOutputSection.h

```diff
@@ -240,8 +240,9 @@ OutputSection<ELFT>::SectionInfo::symbol(SymbolTable<ELFT> &Symbols,
     static auto PrivateSymbolCount = 0U;
 
     auto Name = Generated.add(".LR" + std::to_string(PrivateSymbolCount++));
-    Symbol_ = Symbols.insertSymbol(Name, Section_, Offset_, 0 /*size*/,
-                                   pstore::repo::linkage_type::internal);
+    Symbol_ =
```

Comment:
This change seems to be unnecessary reformatting. Am I missing something?

Reply:
No, your are right. Fixed it.